.github/workflows/performance.yml

```diff
@@ -0,0 +1,26 @@
+name: Build Performance
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-time:
+    name: Build Time Performance
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

Comment:
I think it's failing because we need 14.x node version

Reply:
![image](https://user-images.githubusercontent.com/67006255/173498037-1bc7b9a5-39b5-403d-833c-7fff8bace2e1.png)

Reply:
we already on node 14 version